src/main/webapp/app/exam/participate/exam-start-information/exam-start-information.component.ts

```diff
@@ -0,0 +1,86 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { ArtemisSharedModule } from 'app/shared/shared.module';
+import { ArtemisSharedComponentModule } from 'app/shared/components/shared-component.module';
+import { InformationBox, InformationBoxComponent } from 'app/shared/information-box/information-box.component';
+import { Exam } from 'app/entities/exam.model';
+import { StudentExam } from 'app/entities/student-exam.model';
+import { ArtemisExamSharedModule } from 'app/exam/shared/exam-shared.module';
+import dayjs from 'dayjs/esm';
+
+@Component({
+    selector: 'jhi-exam-start-information',
+    standalone: true,
+    imports: [ArtemisSharedModule, ArtemisSharedComponentModule, InformationBoxComponent, ArtemisExamSharedModule],
+    templateUrl: './exam-start-information.component.html',
+})
+export class ExamStartInformationComponent implements OnInit {
+    examInformationBoxData: InformationBox[] = [];
+
+    @Input() exam: Exam;
+    @Input() studentExam: StudentExam;
+
+    totalPoints?: number;
+    totalWorkingTimeInMinutes?: number;
+    moduleNumber?: string;
+    courseName?: string;
+    examiner?: string;
+    numberOfExercisesInExam?: number;
+    examinedStudent?: string;
+    startDate?: dayjs.Dayjs;
+    gracePeriodInMinutes?: number;
+
+    ngOnInit(): void {
+        this.totalPoints = this.exam.examMaxPoints;
+        this.totalWorkingTimeInMinutes = Math.floor(this.exam.workingTime! / 60);
+        this.moduleNumber = this.exam.moduleNumber;
+        this.courseName = this.exam.courseName;
+        this.examiner = this.exam.examiner;
+        this.numberOfExercisesInExam = this.exam.numberOfExercisesInExam;
+        this.examinedStudent = this.studentExam.user?.name;
+        this.startDate = this.exam.startDate;
+        this.gracePeriodInMinutes = Math.floor(this.exam.gracePeriod! / 60);
+
+        this.prepareInformationBoxData();
+    }
+
+    prepareEachInformationBox(boxTitle: string, boxContent: string | number, boxContentComponent?: string): InformationBox {
```

Comment:
I do not get what the `each` in the method name refers to? Might `getInformationBox` or `buildInformationBox` be a better name?

Reply:
Normally, we build here information boxes next to each other. `each` refers building each information box. But I think buildInformationBox will be a better fit 👍